DeliveryServiceEF/DeliveryServiceEF.Domain/Models/User.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DeliveryServiceEF.Domain.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DeliveryServiceEF.Domain.Models
+{
+    public class User : BaseModel
+    {
+        public string PhoneNumber { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string Name { get; set; }
```

Comment:
maybe split Name into FirstName and LastName ?

Reply:
I will fix it in next PR